README.md

```diff
@@ -105,22 +105,23 @@ USAGE
 # Command Topics
 
 * [`dvc alias`](docs/alias.md) - Manage repository variable aliases.
+* [`dvc autocomplete`](docs/autocomplete.md) - display autocomplete installation instructions
 * [`dvc cleanup`](docs/cleanup.md) - Replace a DevCycle variable with a static value in the current version of your code. Currently only JavaScript is supported.
 * [`dvc diff`](docs/diff.md) - Print a diff of DevCycle variable usage between two versions of your code.
 * [`dvc environments`](docs/environments.md) - Create a new Environment for an existing Feature.
-* [`dvc features`](docs/features.md) - Access or modify Features with the Management API.
+* [`dvc features`](docs/features.md) - Create, view, or modify Features with the Management API.
 * [`dvc generate`](docs/generate.md) - Generate Devcycle related files.
 * [`dvc help`](docs/help.md) - Display help for dvc.
 * [`dvc keys`](docs/keys.md) - Retrieve SDK keys from the Management API.
 * [`dvc login`](docs/login.md) - Log in to DevCycle.
 * [`dvc logout`](docs/logout.md) - Discards any auth configuration that has been stored in the auth configuration file.
 * [`dvc org`](docs/org.md) - Switch organizations.
-* [`dvc projects`](docs/projects.md) - Access Projects with the Management API.
+* [`dvc projects`](docs/projects.md) - View Projects with the Management API.
```

Comment:
since there's a `projects create` command, should we mention that you can create a project?

Reply:
yep, good call